frontend/src/e2e-playwright/pages/citizen/citizen-pedagogical-documents.ts

```diff
@@ -46,4 +48,16 @@ export default class CitizenPedagogicalDocumentsPage {
   async downloadAttachment(id: string) {
     await this.#downloadAttachment(id).click()
   }
+
+  async assertChildNameIsNotShown(id: string) {
+    await waitUntilFalse(() => this.#childName(id).isVisible())
```

Comment:
JS dependencies were updated yesterday, and with the new Playwright version you could now use `await this.#childName(id).waitFor('hidden')` if you want.